### PR TITLE
[AdminBundle] Added up/down arrows on nested forms, fix rich editor when moving elements in DOM

### PR DESCRIPTION
**Pull request by sandergo90**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | #865 #216

Nested forms can now also be sorted by using the up and down arrow.

Also a fix for the rich editor. When moving a element around in the DOM by sorting it, the ckeditor gets disabled and the content is removed. This is a known bug in Ckeditor. I have fixed it by destroying all the rich editors and reinitialise them when moving them around.

### CODE REVIEW

File: src/Kunstmaan/AdminBundle/Resources/ui/js/_pagepart-editor.js [hunk 1]
```diff
@@ -293,6 +305,28 @@ kunstmaanbundles.pagepartEditor = function (window) {
             cb({target: target})
         })
     };
+    reInit = function($el) {
+        $($el).find('*[data-reinit-js]').each(function() {
```
Comment (diskwriter) [inline src/Kunstmaan/AdminBundle/Resources/ui/js/_pagepart-editor.js]: $el is already a jQuery element here

File: src/Kunstmaan/AdminBundle/Resources/ui/js/_pagepart-editor.js [hunk 2]
```diff
@@ -184,6 +184,9 @@ kunstmaanbundles.pagepartEditor = function (window) {
 
         var currentPp = $('#' + targetId + '-pp-container');
```
Comment (diskwriter) [inline src/Kunstmaan/AdminBundle/Resources/ui/js/_pagepart-editor.js]: Could you add the $var syntax for these variables?

File: src/Kunstmaan/AdminBundle/Resources/ui/js/_pagepart-editor.js [hunk 3]
```diff
@@ -208,6 +214,9 @@ kunstmaanbundles.pagepartEditor = function (window) {
...
```
Comment (diskwriter) [inline src/Kunstmaan/AdminBundle/Resources/ui/js/_pagepart-editor.js]: Could you add the $var syntax for these variables?

**Comment by sandergo90** [comment]
@diskwriter good catch, done.

**Pull request merged**